<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caculator</class>
 <widget class="QMainWindow" name="Caculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="display">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="result">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>136</y>
      <width>358</width>
      <height>236</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_sin">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>sin(</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="Button_pow_2">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>pow(</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="Button_point">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Button_sqrt">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>sqrt(</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="Button_1">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Button_tan">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>tan(</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="Button_3">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Button_cos">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>cos(</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="Button_pow">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="Button_0">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="Button_6">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="Button_2">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="Button_5">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Button_7">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Button_mod">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Button_9">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="Button_divided">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="Button_times">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QPushButton" name="Button_left">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QPushButton" name="Button_right">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="Button_delete">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="Button_comma">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="Button_8">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="Button_minus">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="Button_4">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="Button_plus">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="2">
      <widget class="QPushButton" name="Button_equal">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>=</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
